### DISCUSSION TITLE
Play metronome sound when opening timing mode in editor

### DISCUSSION THREAD

**Discussion by papertek** [ppy/osu#22575] [Ideas] [answered]
It feels weird to not have the metronome play when first entering timing mode (compared to stable).

https://user-images.githubusercontent.com/72106503/217741595-e69f55ee-5e6d-4e36-b250-abbdc2a597c8.mp4

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/22597
